generators/generator-bot-core-qna/README.md

```diff
@@ -1,42 +1,30 @@
 # @microsoft/generator-bot-core-qna [![NPM version][npm-image]][npm-url]
 
-This template guides you through creating and connecting a bot to a QnA Maker knowledge base. You can connect to an existing knowledge base, create one from scratch using .qna files, or create one from an existing FAQ-style website.
+A simple question-and-answer bot with Azure QnA Maker
 
-## What this template is for
+### Recommended use
 
-Use this template if you want to...
+- Create a simple question-and-answer bot with Azure QnA Maker
+- Customize and extend question-and-answer pairs or connect to your website's FAQ
+- Extend your bot with [Azure Bot Framework components](https://aka.ms/ComponentTemplateDocumentation)
 
-- Create a bot and connect it to a QnA Maker knowledge base
+### Included capabilities
 
-## Packages
+- Answer questions from a QnA Maker knowledgebase
 
-Your bot can use the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend the base functionality. From Composer, use the Package Manager to discover additional packages you can add to your bot.
+### Required Azure resources
 
-## Languages
+- [Azure Language Understanding (LUIS)][luis], or another recognizer of your choice
+- [Azure QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
 
-- English (en-US)
-
-## Azure Resource Deployment
-
-To run this bot you'll need the resources listed below. Create a publishing profile in Composer to provision and publish to your Azure resources for your bot.
-
-- [LUIS][luis], or another recognizer of your choice
-- [QnA Maker](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/overview/overview)
-
-## Using this template
+### Supported languages
 
-### From Composer
-
-From Composer you'll use the **New** button on the **Home** screen to create a new bot. After creation, Composer will guide you through making customizations to your bot. If you'd like to extend your bot with code, you can open up your bot using your favorite IDE (like Visual Studio) from the location you choose during the creation flow.
-
-### From the command-line
-
-This template can also be installed from the [command line](https://github.com/microsoft/botframework-components/blob/main/generators/command-line-instructions).
+- English (en-US)
 
-## License
+### License
 
 [MIT License](https://github.com/microsoft/botframework-components/blob/main/LICENSE)
 
 [luis]: https://docs.microsoft.com/en-us/azure/cognitive-services/luis/what-is-luis
-[npm-image]: https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-core-qna.svg
-[npm-url]: https://www.npmjs.com/package/@microsoft/generator-bot-core-qna
+[npm-image]: https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-core-assistant.svg
```

Comment:
These should point to generator-bot-core-qna

Reply:
nice catch.  I actually had this copy pasta bug in a few other places.  I've updated this and the others I found.